deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinCollection.java

```diff
@@ -112,7 +114,6 @@
      */
     public TwinCollection()
     {
-        /* SRS_TWIN_COLLECTION_21_001: [The constructor shall create a new instance of the super class.] */
```

Comment:
Sweeping comment for this type of removal. Are these comments no longer important to the SDK?

Reply:
Yeah, we've stopped using these lines to correlate from production code to unit tests

Reply:
Since they are often out of date, I've been deleting them over time so they stop cluttering up the code